caikit/runtime/http_server.py

```diff
@@ -106,15 +106,18 @@ class RuntimeHTTPServer(RuntimeServerBase):
     ## Interface ##
     ###############
 
-    def __init__(self, tls_config_override: Optional[aconfig.Config] = None):
+    def __init__(
+        self,
+        inference_service: ServicePackage,
+        #  # pylint: disable=unused-argument
```

Comment:
This syntax looks odd

Reply:
Also, if these are `Optional` should they have a default value of `None`?

Reply:
Hmm, I tried to use the same one we have for `grpc_server`?
```
    def __init__(
        self,
        inference_service: ServicePackage,
        training_service: Optional[ServicePackage],
```

Reply:
But yeah the `optional` should probably be None in both places